### ISSUE TITLE
Screen Refreshing Problem - Customer Refunds

### ISSUE THREAD

**Issue by i-rabot** [kbower/tickettest1#3355]
_Imported from trac ticket 3355.
Created by **jobrien**
Opened in trac: Sep 2, 2015, 12:19:35 PM
Last modified in trac: Nov 4, 2015, 2:05:31 PM_
***
_by jobrien:_
When doing a second, third, etc. refund in Customers, perhaps only after doing a finance refund - the type of refund dropdown box is hidden behind the screen.  You can type in a letter like "f" and in that case "finance" would be selected and populated in the box.  If you close the refund box and reopen, the dropdown then appears as expected.

Sue

[[Image(jchahebb.png)]]

***
* sjohnson attached [jchahebb.png](https://trac.retailarchitects.com/trac/attachment/ticket/3355/jchahebb.png) on Sep 2, 2015, 12:19:35 PM

**Comment by i-rabot** [comment]
_**kbower** trac comment on Sep 2, 2015, 12:22:35 PM:_

This "smells" like something binding would fix instead of functional programming trying to catch all the cases it should or shouldn't be showing...

**Comment by i-rabot** [comment]
_**kbower** trac comment on Oct 14, 2015, 12:38:58 PM:_

Scott suspects this is a duplicate of [#3401](3401) and I agree.  The screen is quite flaky and behaves inconsistently depending on previous use.

Susan to test whether this is fixed with [#3401](3401).

**Comment by i-rabot** [comment]
_**sjohnson** trac comment on Nov 4, 2015, 2:04:36 PM:_

[[6845]](../commit/2a298e23573149b97644d1d86a7905764858b631) did not fix this issue.  Slumberland retested today(11/4/15) and it is still occurring.
